### ISSUE TITLE
checkbox complains about the [common] section in canonical-certification.conf

### ISSUE THREAD

**Issue by bladernr** [canonical/checkbox#1149]
### Bug Description

Checkbox is complaining about the [common] header section in canonical-certification.conf.

### To Reproduce

Install 24.04 
install canonical-certification-server from beta channel
run a launcher from that provider (like certify-24.04) and note the warnings once excution begins

### Environment

- noble
- checkbox 4.0.0.dev194

### Relevant log output

```shell
ubuntu@luma:~$ certify-22.04
Unexpected section [common]. Origin: /etc/xdg/canonical-certification.conf
WARNING:plainbox.secure.providers.v1:Skipped file: /usr/share/checkbox-provider-base/units/camera/README.rst
WARNING:plainbox.secure.providers.v1:Skipped file: /usr/share/checkbox-provider-base/units/snapd/README.md
WARNING:plainbox.secure.providers.v1:Skipped file: /usr/share/checkbox-provider-base/units/stress/suspend_cycles_reboot.md
```

### Additional context

It appears to be complaining about the [common] section of the config file which has existed for a very long time... this is separate from the warnings though.

```
[common]
welcome_text = Welcome to System Certification!
 
 This application will gather information from your system. Then you will be asked manual tests to confirm that the system is working properly. Finally, you will be asked for the Secure ID of the computer to submit the information to the certification.canonical.com database.

 Please be sure you have edited /etc/xdg/canonical-certification.conf as necessary to set your Secure ID, and network and virtualization test parameters.

 To learn how to create or locate the Secure ID, please see here:

 https://certification.canonical.com/home/ubuntu-certified
```

**Comment by Hook25** [comment]
I've never heard of the `common` section at all. I have tracked this down to double check and config were refactored in 40733bda6169f4dc112a7608ec41cab5d95e74b8 introducing also a schema. The schema does not include any common section. Just to be sure I went back to 40733bda6169f4dc112a7608ec41cab5d95e74b8~1 (005af7dffa4598f4d3506fc7064b62c343574384) and searched for any mention of this section. I am unable to track down any parser for it in the old baroque config system. I have parsed with `check-config` and xdg config with a common section and this is the result:
```
Configuration files:
 - /etc/xdg/checkbox.conf
   [launcher]
   launcher_version=1
   [launcher]
   app_id=checkbox-cli
   [launcher]
   app_version=unset
   [launcher]
   api_flags=['restartable']
   [launcher]
   api_version=0.99
   [launcher]
   stock_reports=['text', 'certification', 'submission_files']
   [launcher]
   local_submission=True
   [launcher]
   session_title=session title
   [launcher]
   session_desc=
   [test plan]
   filter=['*']
   [test plan]
   unit=unset
   [test plan]
   forced=False
   [test selection]
   forced=False
   [test selection]
   exclude=[]
   [ui]
   type=interactive
   [ui]
   output=show
   [ui]
   dont_suppress_output=False
   [ui]
   verbosity=normal
   [ui]
   auto_retry=False
   [ui]
   max_attempts=3
   [ui]
   delay_before_retry=1
   [daemon]
   normal_user=
   [restart]
   strategy=unset
   [restart]
   [environment]
   [daemon]
   [manifest]
No validation problems found
```

Even though "No validation problems found", the section is not parsed at all. Are you sure this was ever actually used to do anything? If so, what was the effect and when was it last observed?

**Comment by bladernr** [comment]
I honestly don't recall... we were told to include that a very, very long time ago when the config file first came about. My recollection was that it was supposed to be read and presented to the user as some sort of welcome or useful information.  I really don't remember ever even noticing it, but I've run this stuff so many tens of thousands of times over the years and have been running it mostly automatically using launchers that do not allow for user prompts that I could either have missed it or forgotten it, or never noticed and only realized something was amiss with the warnings that were dumped.

I suspect that perhaps this was meant to be used but never fully baked, and then forgotten about and then never really pointed out until recent changes that seem to be a bit  more verbose with the warnings and errors.

Thanks for looking, I think, given this, that the easiest path is just to remove that section then from the file and not worry about it further.
